Lib/test/test_types.py

```diff
@@ -1738,18 +1738,37 @@ def test_constructor(self):
         ns1 = types.SimpleNamespace()
         ns2 = types.SimpleNamespace(x=1, y=2)
         ns3 = types.SimpleNamespace(**dict(x=1, y=2))
+        ns4 = types.SimpleNamespace({'x': 1, 'y': 2}, x=4, z=3)
+        ns5 = types.SimpleNamespace([['x', 1], ['y', 2]], x=4, z=3)
+        ns6 = types.SimpleNamespace(UserDict({'x': 1, 'y': 2}), x=4, z=3)
```

Comment:
Consider adding single-arg-only examples too.

```suggestion
        ns4 = types.SimpleNamespace({'x': 1, 'y': 2}, x=4, z=3)
        ns5 = types.SimpleNamespace([['x', 1], ['y', 2]], x=4, z=3)
        ns6 = types.SimpleNamespace(UserDict({'x': 1, 'y': 2}), x=4, z=3)
        ns7 = types.SimpleNamespace({'x': 1, 'y': 2})
        ns8 = types.SimpleNamespace([['x', 1], ['y', 2]])
```

Maybe also empty-arg examples.

```suggestion
        ns4 = types.SimpleNamespace({'x': 1, 'y': 2}, x=4, z=3)
        ns5 = types.SimpleNamespace([['x', 1], ['y', 2]], x=4, z=3)
        ns6 = types.SimpleNamespace(UserDict({'x': 1, 'y': 2}), x=4, z=3)
        ns7 = types.SimpleNamespace({'x': 1, 'y': 2})
        ns8 = types.SimpleNamespace([['x', 1], ['y', 2]])
        ns9 = types.SimpleNamespace([], x=4, z=3)
        ns10 = types.SimpleNamespace({}, x=4, z=3)
        ns11 = types.SimpleNamespace([])
        ns12 = types.SimpleNamespace({})
```